dataland-frontend/tests/e2e/fixtures/sme/SmePreparedFixtures.ts

```diff
@@ -9,20 +9,18 @@ import { generateSmeFixtures } from "@e2e/fixtures/sme/SmeDataFixtures";
  */
 export function generateSmePreparedFixtures(): Array<FixtureData<SmeData>> {
   const preparedFixtures = [];
-  preparedFixtures.push(manipulateFixtureForYear(generateSmeFixtures(1)[0], 2023));
+  preparedFixtures.push(manipulateFixtureForYear(generateSmeFixtures(1)[0], "2023"));
   return preparedFixtures;
 }
 
 /**
- * Sets the company name, data date and reporting period in the fixture data to
- * specific values needed for tests.
+ * Sets the company name and reporting period in the fixture data to specific values needed for tests.
  * @param input Fixture data to be manipulated
  * @param year the year as a number
  * @returns the manipulated fixture data
  */
-function manipulateFixtureForYear(input: FixtureData<SmeData>, year: number): FixtureData<SmeData> {
-  input.companyInformation.companyName = "SME-year-" + String(year);
-  input.t.financialYear = year;
-  input.reportingPeriod = String(year);
+function manipulateFixtureForYear(input: FixtureData<SmeData>, year: string): FixtureData<SmeData> {
+  input.companyInformation.companyName = "SME-year-" + year;
+  input.reportingPeriod = year;
```

Comment:
@fsoffel  You can check if we even need this anymore

Reply:
It gets used. However the year does not play a large role there. So 🤷 Keep it but remove the year?